rosdoc2/verbs/build/parse_rosdoc2_yaml.py

```diff
@@ -77,14 +61,20 @@ def parse_rosdoc2_yaml(yaml_string, build_context):
         raise ValueError(
             f"Error parsing file '{file_name}', in the second section, "
             f"expected a 'builders' key")
-    builders_dict = config['builders']
-    if not isinstance(builders_dict, dict):
+    builders_list = config['builders']
+    if not isinstance(builders_list, list):
         raise ValueError(
             f"Error parsing file '{file_name}', in the second section, value 'builders', "
-            f"expected a dict{{output_dir: build_settings, ...}}, "
-            f"got a '{type(builders_dict)}' instead")
+            f"expected a list of builders, "
+            f"got a '{type(builders_list)}' instead")
 
     builders = []
-    for output_directory, entry in builders_dict.items():
-        builders.append(parse_builder_entry(output_directory, entry, build_context))
+    for builder in builders_list:
+        if len(builder) != 1:
+            raise ValueError(
+                f"Error parsing file '{file_name}', in the second section, each builder "
+                f"must have a 'type'")
```

Comment:
Is this the right error message?

Reply:
Yeah, that was wrong.  I've cleaned it up in 5b8f2a8.